blind_voting_app/settings_shared.py

```diff
@@ -14,6 +14,17 @@
 BASE_DIR = Path(__file__).resolve().parent.parent
 
 
+try:
+    from .settings_shared import *
+except ImportError:
+    pass
+
+SECRET_KEY = 'my1q)xr9)v7j$ty_113li8e*@6txxll&bn@+=8$po1-qlarq-v'
+
+DEBUG = False
+
+ALLOWED_HOSTS = []
+
```

Comment:
Also, please remove these settings from the shared file please! These belong in secret settings files (`settings_development` and `settings_production`) only!